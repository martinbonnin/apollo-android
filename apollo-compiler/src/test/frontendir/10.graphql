query GetAnimal($condition1: Boolean!) {
  friend {
    ... on Animal @include(if: $condition1){
      ... on Cat  {
        # name1 is not nullable but the enclosing inline fragment should make it nullable
        name1
      }
    }
  }
}
