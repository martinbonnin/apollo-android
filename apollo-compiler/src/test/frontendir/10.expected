package com.example

import kotlin.String

interface GetAnimal {
  interface QueryData {
    val friend: _.AnimalFriend?

    interface AnimalFriend {
      val asCatFriend: _.CatFriend

      interface CatFriend {
        val name1: String?
      }
    }
  }
}
